pkg/apis/core/types_shoot.go

```diff
@@ -308,10 +308,22 @@ type ETCDEncryptionKeyRotation struct {
 	LastInitiationTime *metav1.Time
 	// LastInitiationFinishedTime is the recent time when the ETCD encryption key credential rotation initiation was
 	// completed.
+	//
+	// Deprecated: This field will be removed in a future release. The field will be no longer needed with
+	// the removal `rotate-etcd-encryption-key-start` & `rotate-etcd-encryption-key-complete` annotations.
+	// TODO(AleksandarSavchev): Remove this after support for Kubernetes v1.33 is dropped.
 	LastInitiationFinishedTime *metav1.Time
 	// LastCompletionTriggeredTime is the recent time when the ETCD encryption key credential rotation completion was
 	// triggered.
 	LastCompletionTriggeredTime *metav1.Time
+	// AutoCompleteAfterPrepared indicates whether the current ETCD encryption key rotation should be auto completed after prepared status.
```

Comment:
I think that this wording makes more sense if read by a person who is not familiar with the implementation details.

```suggestion
	// AutoCompleteAfterPrepared indicates whether the current ETCD encryption key rotation should be auto completed after the preparation phase has finished.
```